### PR TITLE
WT-6715 Document the behaviour of logged/non-logged tables

### PR DESCRIPTION
**Pull request by raviprakashgiri29**

### CODE REVIEW

File: src/docs/schema.dox [hunk 1]
```diff
@@ -362,4 +362,16 @@ Here is another example program, @ex_ref{ex_call_center.c}.
 @snippet ex_call_center.c call-center decl
 @snippet ex_call_center.c call-center work
 
+@section logged_non_logged_tables Logged and Non-logged tables
+
+- WiredTiger writes records to the log for each transaction when logging is enabled
+  using the log=(enabled) configuration to wiredtiger_open.
```
Comment (alisonfel) [inline src/docs/schema.dox:368]: np: To help distinguish them as code/configs, might be worth formatting `log=(enabled) ` and `wiredtiger_open` with `\c log=(enabled) ` and `\c wiredtiger_open` (respectively).

**Review by alisonfel — APPROVED** [review]
LGTM, just a small formatting suggestion

File: src/docs/schema.dox (hunk 1 above)
Comment (raviprakashgiri29) [reply src/docs/schema.dox]: Thanks, @alisonfel for flagging, updated the comments.

File: src/docs/schema.dox [hunk 2]
```diff
@@ -362,4 +362,16 @@ Here is another example program, @ex_ref{ex_call_center.c}.
 @snippet ex_call_center.c call-center decl
 @snippet ex_call_center.c call-center work
 
+@section logged_non_logged_tables Logged and Non-logged tables
+
+- WiredTiger writes records to the log for each transaction when logging is enabled
```
Comment (kommiharibabu) [inline src/docs/schema.dox:367]: How about something like the following?
`WiredTiger writes records to the log for each transaction commit when logging is enabled on the tables`

File: src/docs/schema.dox [hunk 3]
```diff
@@ -362,4 +362,16 @@ Here is another example program, @ex_ref{ex_call_center.c}.
...
+  using the \c log=(enabled) configuration to \c wiredtiger_open.
+- WiredTiger supports checkpoint durability on non-logged tables and logging
+  is disabled by default. See @ref checkpoint for information on checkpoint durability.
+- WiredTiger supports commit-level durability on logged tables. See @ref durability for
+  information on commit-level durability.
+- Rollback to stable does not operate on logged tables as the updates on these
```
Comment (kommiharibabu) [inline src/docs/schema.dox:373]: How about adding section or something to indicate that the other differences between the logged tables and non-logged tables? and then mentioning the following two points would be good.

**Review by kommiharibabu — COMMENTED** [review]
Looks good, some more additional details would be good.

File: src/docs/schema.dox (hunk 2 above)
Comment (raviprakashgiri29) [reply src/docs/schema.dox:367]: Fixed

File: src/docs/schema.dox (hunk 3 above)
Comment (raviprakashgiri29) [reply src/docs/schema.dox:376]: Added a different section for the last two points.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**